src/j1client.js

```diff
@@ -10,6 +10,19 @@ const fetch = require("node-fetch").default;
 
 const J1_USER_POOL_ID_PROD = "us-east-2_9fnMVHuxD";
 const J1_CLIENT_ID_PROD = "1hcv141pqth5f49df7o28ngq1u";
+const REQUEST_TIMEOUT_IN_MS = 1000 * 60 * 5; // 5 minute timeout.
+
+const JobStatus = {
+  IN_PROGRESS: "IN_PROGRESS",
+  COMPLETED: "COMPLETED",
+  FAILED: "FAILED"
+};
+
+const sleep = function(ms) {
```

Comment:
I'm not sure what other people's opinions on this are, but I'd prefer `function sleep(ms)` instead of `const sleep = function(ms)`. If you disagree and want to use `const`, you can simply assign to an arrow function (i.e. `const sleep = () => {...}`).

Reply:
Thanks @fomentia I agree with the approach.